.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: csharp
+mono: none
+dotnet: 2.1.4
```

Comment:
Can we filter this down to the same subset of branches that our appveyor.yml file expresses?

Reply:
I pushed a fix for this myself.